shared/simplestreams/products.go

```diff
@@ -25,7 +25,7 @@ type Product struct {
 	Aliases         string                    `json:"aliases"`
 	Architecture    string                    `json:"arch"`
 	OperatingSystem string                    `json:"os"`
-	LXDRequirements map[string]string         `json:"lxd_requirements,omitempty"`
```

Comment:
How will the requirements listed in the `lxd_requirements` field from the `ubuntu:` remote get handled now?

Reply:
oops that's a typo sorry, I don't know why this went away